### PR TITLE
43 - Sample external API call to render results in the browser.

### PR DESCRIPTION
**Pull request by alexventuraio**
Sample to get started with the `fetch`  tool to make API calls.

Screen Shot 2021-10-02 at 23 58 07

### CODE REVIEW

**Comment by swapnilsparsh** [comment]
@alexventuraio Thank you for contributing to the project.
Please resolve the conflicts and add your project to the official website also. 🔥

File: index.html [hunk 1]
```diff
@@ -264,9 +253,14 @@ <h4>Rock Paper Scissors Game</h4>
         <img src="30DaysOfJavaScript/assets/37.png" alt="Rock Paper Scissors Game" />
       </a>
     </div>
+    <div class="item">
+      <a target="_blank" href="39 - Getting Started with APIs/index.html">
+        <h4>Getting Started with APIs</h4>
+        <img src="30DaysOfJavaScript/assets/39.png" alt="Getting Started with APIs" />
+      </a>
+    </div>
```
Comment (alexventuraio) [inline index.html:261]: @swapnilsparsh it already is in the official web page

**Comment by alexventuraio** [comment]
@swapnilsparsh Conflicts already fixed!

**Comment by swapnilsparsh** [comment]
@alexventuraio Please resolve the conflicts.

**Comment by alexventuraio** [comment]
> @alexventuraio Please resolve the conflicts.

@swapnilsparsh done! :)

**Comment by swapnilsparsh** [comment]
@alexventuraio Please resolve the conflicts and update the project number.😅

**Comment by alexventuraio** [comment]
Jajaja ok

> @alexventuraio Please resolve the conflicts and update the project number.😅

jajaja oh my gosh, I assume I would get more conflicts as you merge any PRs, I'll keep try following you 😱

**Comment by swapnilsparsh** [comment]
> Jajaja ok
> 
> > @alexventuraio Please resolve the conflicts and update the project number.😅
> 
> @swapnilsparsh  jajaja oh my gosh, I assume I would get more conflicts as you merge any PRs, I'll keep try following you 😱

No more conflict maybe 😅

**Comment by swapnilsparsh** [comment]
@alexventuraio Thank you for your contribution. Looking forward to more such amazing contributions 🥂

**Comment by alexventuraio** [comment]
> @alexventuraio Thank you for your contribution. Looking forward to more such amazing contributions 🥂

Ok, I'll keep fixing conflicts as you merge them, I'll continue with the rest of them right now

**Pull request merged**